src/components/storagePools/storageVolumeCreateBody.jsx

```diff
@@ -65,15 +65,20 @@ const VolumeDetails = ({ idPrefix, size, unit, format, storagePoolCapacity, stor
     }
 
     if (validVolumeFormats) {
-        formatRow = (
-            <FormGroup fieldId={`${idPrefix}-fileformat`} label={_("Format")}>
-                <FormSelect id={`${idPrefix}-format`}
-                    onChange={(_event, value) => onValueChanged('format', value)}
-                    value={format}>
-                    { validVolumeFormats.map(format => <FormSelectOption value={format} key={format} label={format} />) }
-                </FormSelect>
-            </FormGroup>
-        );
+        if (!format)
+            console.error("VolumeDetails internal error: format is not set for storage pool type", storagePoolType);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1327-20231124-094238-e70920c2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Annotated locally, will push with the next round.